### DISCUSSION TITLE
Go viral automation

### DISCUSSION THREAD

**Discussion by Rambooo69** [atalaydenknalbant/autosubscriber-youtube#25] [General]
Ever since go viral has been removed, I have been using go viral directly in there site but i noticed it keeps freezing and only works if i refresh the page. I didn't have this problem when using your script.. do u think u can make a scrip where its only job is to make sure the earning is running and dose nothing else so we don't get banned..?

**Comment by atalaydenknalbant** [comment]
I tried what you said but even random wait times this website detects you. I will try once more with undetecteble chrome driver but later.

**Reply by Rambooo69** [reply]
Ok
